profiling/default.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs,config }:
+let
+ name = "hc-sim2h-server-perf";
+  script-sim2h-server-perf = pkgs.writeShellScriptBin name
+  ''
+  cd crates/sim2h_server && cargo build -p sim2h_server --release && perf record --call-graph dwarf sim2h_server
+  '';
+
+  script-hc-conductor-perf = pkgs.writeShellScriptBin "hc-conductor-perf"
+  ''
+  cd crates/holochain && cargo build -p holochain --release && perf record --call-graph dwarf holochain
+  '';
+
+  script-hc-generate-flame-graph = pkgs.writeShellScriptBin "hc-generate-flame-graph"
+  ''
+  rm -rf FlameGraph && git clone https://github.com/brendangregg/FlameGraph && perf script | perl FlameGraph/stackcollapse-perf.pl | perl FlameGraph/flamegraph.pl > generated-graph.svg
```

Comment:
Is there a reason we need to download the repo every time we generate a flame graph?  This also means we can run this command when we we aren't online.

Reply:
was looking at getting the latest version of the repo everytime but will change this around